src/AutomationTests/TargetElementLocatorUnitTests.cs

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-using System.Collections.Generic;
-using Axe.Windows.Automation;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-
-namespace Axe.Windows.AutomationTests
-{
-    [TestClass]
-    public class TargetElementLocatorUnitTests
-    {
-        [TestMethod]
-        [Timeout (1000)]
-        public void LocateElement_NoTargetSpecifiedInParameters_ThrowsAutomationException_ErrorAutomation007()
-        {
-            try
-            {
-                TargetElementLocator.LocateRootElement(-1);
-            }
-            catch (AxeWindowsAutomationException ex)
-            {
-                Assert.IsTrue(ex.Message.Contains("Automation017:"));
```

Comment:
>Automation017 [](start = 51, length = 13)

Is there (or should there be) a replacement unit test to confirm the Automation017 message?

Reply:
All the unit tests are changing.